File: Python/priorityQueue/People.py

class People:
    def __init__(self, name, priority):
        self.name = name
        self.priority = priority

    def __str__(self):
        return self.name + " " + str(self.priority)

    def getName(self):
        return self.name

    def getPriority(self):
        return self.priority

File: java/src/main/java/org/example/priorityqueue/People.java
package org.example.priorityqueue;

public class People {
    private String name;
    private int priority;

    public People(String name, int priority) {
        this.name = name;
        this.priority = priority;
    }

    public String getName() {
        return name;
    }

    public int getPriority() {
        return priority;
    }
}
